time-elements.js

```diff
@@ -561,7 +563,9 @@
     var options = {
       hour: el.getAttribute('hour'),
       minute: el.getAttribute('minute'),
-      second: el.getAttribute('second')
+      second: el.getAttribute('second'),
+      timeZone: el.getAttribute('timeZone'),
+      timeZoneName: el.getAttribute('timeZoneName')
```

Comment:
> You think we should take it out again?

Yes, I think we should focus solely on the original purpose of this PR: making it explicit to the user which timezone the timestamp is rendered in (always theirs). Proposal:

```html
<local-time> → 8 Apr 2018 21:30
<local-time timezone="short"> → 8 Apr 2018 21:30 CEST
<local-time timezone="numeric"> → 8 Apr 2018 21:30 +02:00
```